es-app/src/guis/GuiInfoPopup.cpp

```diff
@@ -0,0 +1,114 @@
+#include "guis/GuiInfoPopup.h"
+#include "Renderer.h"
+#include "components/TextComponent.h"
+#include "Log.h"
+
+GuiInfoPopup::GuiInfoPopup(Window* window, std::string message, int duration) :
+	GuiComponent(window), mMessage(message), mDuration(duration), running(true)
+{
+	mFrame = new NinePatchComponent(window);
+	float maxWidth = Renderer::getScreenWidth() * 0.9f;
+	float maxHeight = Renderer::getScreenHeight() * 0.2f;
+
+	std::shared_ptr<TextComponent> s = std::make_shared<TextComponent>(mWindow,
+		"",
+		Font::get(FONT_SIZE_MINI),
+		0x444444FF,
+		ALIGN_CENTER);
+
+	// we do this to force the text container to resize and return an actual expected popup size
+	s->setSize(0,0);
+	s->setText(message);
+	mSize = s->getSize();
+
+	// confirm the size isn't larger than the screen width, otherwise cap it
+	if (mSize.x() > maxWidth) {
+		s->setSize(maxWidth, mSize[1]);
+		mSize[0] = maxWidth;
+	}
+	if (mSize.y() > maxHeight) {
+		s->setSize(mSize[0], maxHeight);
+		mSize[1] = maxHeight;
```

Comment:
Ok, you're going to hate me for this...

When I replace the warning text about missing auto-system theme nodes in the xml, with something ridiculously long, such as the two first paragraphs of Lorem Ipsum...., I get this:

![debugging_autosystems2](https://user-images.githubusercontent.com/6103768/27932082-aee2974e-629c-11e7-8788-2f4c43b25ea6.png)
This is on a 1024x768 windowed screen, but that should not matter, as all sizes are in relative screen sizes anyways. As you can see, the textComponent is placed centered on the screen, basically sticking out on the top and bottom. 
Now, I did not expect this to go completely right, but then again, I was not expecting this either. First of all, it overrides the top-bottom padding, second, the centering obscures the top part.

We should consider an implementation like the description box in the detailed gamelistview, which is a textComponent, within a scrollableContainerComponent.

The expected / optimal behavior for me would be a popup window, respecting the paddings, that allows a user to scroll through the text using up-down or similar.

Now, do you need to fix this now? No, probably not, seeing as the only instance of this is hard-coded by yourself. But this PR is the introduction of this new functionality, so hey, I thought, lets test it. 
I am also fine with it if you want me to make a separate issue out of it, to be fixed at a later date. 

Actually, with the refactoring of the GameListViews, there will be a ScrollableTextComponent, which might be more suitable to use, so maybe we better wait for that to land. 

(Argh! All these dependencies: lets wait for this, and this, and this, like a reverse bootstrapping, boot-undoing? Barrel full of crab?)